blobstore/src/main/java/org/jclouds/blobstore/config/LocalBlobStore.java

```diff
@@ -364,6 +364,9 @@ private SortedSet<StorageMetadata> extractCommonPrefixes(SortedSet<StorageMetada
             o = prefix + o;
          }
          md.setName(o + delimiter);
+         if (!contents.contains(md)) {
+            contents.add(md);
+         }
          contents.add(md);
```

Comment:
Doesn't this add `md` twice?

Reply:
This was a typo and I removed it. Thanks!